autoload/lightline/colorscheme/wombat.vim

```diff
@@ -14,20 +14,21 @@ let s:base1 = [ '#969696', 247 ]
 let s:base2 = [ '#a8a8a8', 248 ]
 let s:base3 = [ '#d0d0d0', 252 ]
 let s:yellow = [ '#cae682', 180 ]
-let s:orange = [ '#e5786d', 173 ]
+let s:magenta = [ '#f289ff', 173 ]
 let s:red = [ '#e5786d', 203 ]
-let s:magenta = [ '#f2c68a', 216 ]
+let s:orange = [ '#f2c68a', 216 ]
 let s:blue = [ '#8ac6f2', 117 ]
 let s:cyan = s:blue
 let s:green = [ '#95e454', 119 ]
-let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
+let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'special': {}, 'tabline': {}}
 let s:p.normal.left = [ [ s:base02, s:blue ], [ s:base3, s:base01 ] ]
 let s:p.normal.right = [ [ s:base02, s:base0 ], [ s:base1, s:base01 ] ]
 let s:p.inactive.right = [ [ s:base023, s:base01 ], [ s:base00, s:base02 ] ]
 let s:p.inactive.left =  [ [ s:base1, s:base02 ], [ s:base00, s:base023 ] ]
 let s:p.insert.left = [ [ s:base02, s:green ], [ s:base3, s:base01 ] ]
 let s:p.replace.left = [ [ s:base023, s:red ], [ s:base3, s:base01 ] ]
-let s:p.visual.left = [ [ s:base02, s:magenta ], [ s:base3, s:base01 ] ]
+let s:p.special.left = [ [ s:base023, s:magenta ], [ s:base3, s:base01 ] ]
+let s:p.visual.left = [ [ s:base02, s:orange ], [ s:base3, s:base01 ] ]
```

Comment:
This commit adds support for the new special highlight to all color schemes. The new mode is generally similar to `replace` mode (a bit lighter). In few cases I made some adjustments (no apparent changes to previous colors, though).